recipes/stages/classification/class_incr_multilabel.yaml

```diff
@@ -0,0 +1,37 @@
+_base_: [
+    './train_multilabel.yaml',
+    '../_base_/data/cls_data.py',
+    '../_base_/models/classifiers/classifier.py',
+]
+
+runner:
+    max_epochs: 50
+
+optimizer_config:
+    type: SAMOptimizerHook
+
+optimizer:
+    type: SGD
+    lr: 0.007
+    momentum: 0.9
+    weight_decay: 0.0005
+
+evaluation:
+    metric: ['accuracy', 'class_accuracy']
+
+task_adapt:
+    type: 'mpa'
+    op: 'REPLACE'
+
+custom_hooks:
+- type: ModelEmaV2Hook
+- type: LazyEarlyStoppingHook
```

Comment:
Please remove early-stop related configs. Early stop is working as learning parameters, so we can handle early stop in template.yaml
More details: https://github.com/openvinotoolkit/training_extensions/blob/develop/external/model-preparation-algorithm/mpa_tasks/apis/task.py#L205